examples/xdp-hello/xdp-hello/Cargo.toml

```diff
@@ -5,8 +5,8 @@ edition = "2021"
 publish = false
 
 [dependencies]
-aya = { version = ">=0.11", features = ["async_tokio"] }
-aya-log = "0.1"
+aya = { git = "https://github.com/aya-rs/aya", branch = "main", features = ["async_tokio"] }
```

Comment:
I think `branch = "main"` can be removed here and everywhere; we should use whatever the default branch is (which is defined by the server).